src/runtime_src/driver/include/ert.h

```diff
@@ -109,6 +109,25 @@ struct ert_start_kernel_cmd {
   uint32_t data[1];          /* count-1 number of words */
 };
 
+#define KDMA_UNIT   64
```

Comment:
My preference would be to keep a much simpler user exposed ert_kdma_cmd, which just allows opcode, srcbo, dstbo, offsets and size.   The scheduler can repopulate this command in place with a new opcode and proper bo addresses and cumasks.